var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -116,6 +116,7 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
     variant("pytorch", default=False, description="Enable libtorch support")
     variant("quip", default=False, description="Enable quip support")
     variant("mpi_f08", default=False, description="Use MPI F08 module")
+    variant("smeagol", default=False, description="Enable libsmeagol support", when="@2025.2:")
```

Comment:
wonder if we should have when="@master:".

Reply:
My understanding is that since `2025.2` doesn't exist yet, this has the same effect (i.e. only works with `cp2k@master`) but has the advantage that we don't have to change it from `master` to `2025.2` when adding the new version. I might be wrong though, I'll do a proper test tomorrow.

Reply:
@RMeli your understanding is correct, see https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison ("infinity versions").